Pipfile

```diff
@@ -6,6 +6,7 @@ verify_ssl = true
 [dev-packages]
 pylint = "*"
 pdoc3 = "*"
```

Comment:
should pdoc3 be removed?

Reply:
Since it's under dev-packages, so it doesn't really affect anything.  pdoc3 is pretty much plug-n-play and sphinx requires more configuration.  I figured I am leaving it here at the boilerplate so people can try it out if they want? I'm also open to removing it tho. /shrugs